src/main/java/org/motechproject/newebodac/exception/EntityIsUsedInAnotherEntityException.java

```diff
@@ -0,0 +1,14 @@
+package org.motechproject.newebodac.exception;
+
+public class EntityIsUsedInAnotherEntityException extends BaseFormattedException {
```

Comment:
can you change the name of this exception, maybe "RelationViolationException" or something similar